principles/3_Privacy.md

```diff
@@ -0,0 +1,51 @@
+# Privacy
+
+## Data Minimization
+
+* The minimum amount of data possible shall be collected, for the purposes for which it is gathered. Data that is not used in the disclosure to healthcare authorities should not be collected unless it is essential to the purposes of the application
+* Users can choose how much data to share from symptom surveys
+
+## Consent
+* A member of the public (MoP) shall consent to data being collected or transmitted explicitly, that is it should not be implied by usage, or mixed with another consent
+* A MoP shall provide informed consent, that is they must be able to understand what they are consenting to and the purposes for which each category of data is gathered
+* Information about privacy shall be communicated in plain English, not legal, or technical language
```

Comment:
Suggested edit for clarity:
"Information about privacy shall be communicated in as simple language as possible while maintaining clarity. Specifically, legal or technical language should be avoided."

I guess my gripe with "plain English" is that that could still include fancy words that are not commonly known, also that English might not be the relevant language.

Reply:
Also not all users are english speakers. The "plain language" term sounds better to me.
Small concern that "plain english" may have some legal weight that "plain language" doesn't?

Reply:
Very good points, I have amended as proposed.  I don't think plain english has any legal standing.